modules/statistics/php/statistics_dd_site.class.inc

```diff
@@ -62,14 +57,8 @@ class Statistics_DD_Site extends statistics_site
                 intval($_REQUEST['CenterID'])
             );
         } else {
-            // For the short term the user we'll be granted access
+            // FIXME (#6742): For the short term the user we'll be granted access
             // if at least have permission AND one of the centers
```

Comment:
```suggestion
            // if they have permission and belong to ANY of the centers
```
Edit to clarify what the issue is -- the problematic part is that any center affiliation get them all centers.